.wasm/.clippy.toml

```diff
@@ -0,0 +1,25 @@
+avoid-breaking-exported-api = false
+disallowed-methods = [
+    # https://github.com/serde-rs/json/issues/160
+    "serde_json::from_reader",
+]
+disallowed-types = [
+    "std::collections::HashMap",
+    "std::collections::HashSet",
+    { path = "std::time::Duration", reason = "Use web-time to return a Duration that works under WASM" },
```

Comment:
It turns out that `Duration` is always the one from `std::time` so this gives false positives as well. We could check for `UNIX_EPOCH` however.